src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs

```diff
@@ -84,19 +82,19 @@ public override async Task<int> ExecuteAsync()
             }
         }
 
-        private async Task PromoteBuildAsync(int PromoteToMaestroChannelId)
+        private async Task PromoteBuildAsync(Build build, Channel targetChannel)
         {
-            if (string.IsNullOrEmpty(_options.AzureDevOpsPat))
-            {
-                LocalSettings localSettings = LocalSettings.LoadSettingsFile(_options);
-                _options.AzureDevOpsPat = localSettings.AzureDevOpsToken;
-            }
+            LocalSettings localSettings = LocalSettings.LoadSettingsFile(_options);
```

Comment:
I don't think this is necessary, as the RemoteFactory will do the parsing of the options and local settings for you,